### PR TITLE
Twitter link updated to new PyDelhiConf twitter page.

### PR DESCRIPTION
**Pull request by shashankaryan**

### CODE REVIEW

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Can you please have a bit.ly link here so that we can track how many people visited it. It's better to ask @theskumar because he has created all links previously. This is to ensure that one person has access to all the links generated :)

**Comment by shashankaryan** [comment]
Okay. I will ask him and update the pr asap.

File: index.html [hunk 1]
```diff
@@ -2179,8 +2179,8 @@ <h4 class="modal-title" id="myModalLabel">Long Vesion Code of Conduct</h4>
                <!-- Social Icons -->
                <div class="footer-social-icons">
                    <a href="http://bit.ly/pydelhi-git" target="_blank"><i class="fa-2x fa fa-github"></i></a>
-                   <a href="http://bit.ly/pydelhi-fb" target="_blank"><i class="fa-2x fa fa-facebook"></i></a>
-                   <a href="http://bit.ly/pydelhi-twitter" target="_blank"><i class="fa-2x fa fa-twitter"></i></a>
+                   <a href="http://bit.ly/pydelhiconf_fb" target="_blank"><i class="fa-2x fa fa-facebook"></i></a>
```
Comment (theskumar) [inline index.html]: Can you add a comment here, that you own the analytics for these links. Rest looks good.

**Review by theskumar — CHANGES_REQUESTED** [review]

**Comment by theskumar** [comment]
@shashankaryan thanks

**Pull request merged**